prov/efa/src/rxr/rxr_ep.c

```diff
@@ -2638,3 +2638,79 @@ void rxr_ep_record_tx_op_completed(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_
 	}
 }
 
+#if EFA_UNIT_TEST
+
+#include "efa_unit_tests.h"
+
+struct efa_resource {
```

Comment:
we have similar structure in `efa_av.c` can be put it in a common place and reuse them?

Reply:
will do.